test/tests/functional/pbs_server_periodic_hook.py

```diff
@@ -106,7 +108,11 @@ def check_next_occurances(self, count, freq,
                                             interval=(hook_run_time + 1),
                                             starttime=search_after)
                 time_logged = self.get_timestamp(msg[1])
-                self.assertFalse((time_logged - time_expected) > 1)
+                if alarm != 0:
+                        self.assertLess(time_logged - time_expected,
+                                        alarm - hook_run_time)
+                else:
+                        self.assertLess(time_logged - time_expected, 1)
```

Comment:
Oops, sorry, should this be assertLessEqual? Just to keep the original assert the same? Or should it actually be assertLess.

Reply:
You are right, since get_timestamp returns an integer, in some cases it could be 1 second apart and that should be acceptable. I guess I should do the same for the other assert as well.

Reply:
Is this a case where we should use the fuzzy assert?  I can't remember the name of it off the top of my head, but you provide X, Y, and a delta where X and Y should be equal within the delta?